sample_config.toml

```diff
@@ -50,7 +50,13 @@ name = "storage shed"
 username = "admin"
 password = "987654321"
 address = "192.168.1.245:9000"
+# The address could instead be set to a camera UID like this
+# address = "ABCD01234567890EFG"
```

Comment:
I would prefer to introduce a new key, `uid`. They are technically separate namespaces - it's not _totally_ [inconceivable](https://www.youtube.com/watch?v=D9MS2y2YU_o) that someone could give `ABCD01234567890EFG` a name in DNS.

Then, we can start recommending people use the uid of their cameras, and only use `address` if they're on a different subnet or like Docker for some reason :)

Reply:
(because you have to admit, the real client's workflow for discovering and adding cameras is pretty slick)

Reply:
What should we do in the event that both uid and addr are given? Error, use addr, use uid?

Reply:
Making it do both in a decent way might require a larger refractor. Ideally I'd want it to connect to both, then drop one like the offical client does. But it's really not set up to do that.

Reply:
Hmm, a good question. The strictest thing to do right now might be to error in that situation - that lets us open up the syntax in the future to allow what you describe, but lets us avoid breaking backward compat with config files written today.

Reply:
Done we error out for now. This is done in the validation. I handle the camera address logic in a `utils.rs` which will helpfully ease the transition when we get to it as it is all in one place for all subcommands